plugins/org.jboss.tools.openshift.ui/plugin.xml

```diff
@@ -235,6 +235,31 @@
 				</visibleWhen>
 			</command>
 		</menuContribution>
+        <menuContribution
+            allPopups="false"
+            locationURI="popup:org.jboss.tools.openshift.express.ui.explorer.expressConsoleViewPopup.new?after=additions">
+            <command
+                commandId="org.jboss.tools.openshift.ui.command.resource.new"
+                style="push"
+                icon="icons/resource.gif"
+                tooltip="Create a new resource">
```

Comment:
I disagree. That gives us room to be able to create resources from a text editor in the wizard, in the future